float32 joint1
float32 joint2
float32 joint3
float32 joint4
float32 joint5
float32 joint6
float32 joint7
int64 kinova_s
---
bool joint_velocity_result